homeassistant/auth/auth_store.py

```diff
@@ -207,18 +207,16 @@ async def async_create_refresh_token(
         self._async_schedule_save()
         return refresh_token
 
-    async def async_remove_refresh_token(
-        self, refresh_token: models.RefreshToken
-    ) -> None:
+    @callback
+    def async_remove_refresh_token(self, refresh_token: models.RefreshToken) -> None:
```

Comment:
what about all other funtions above? they also do nothing async anymore 🤔

Reply:
I was trying to keep the scope small so that we could get this in and then merge your PR. After that we can go back and clean up some more of them.

Reply:
mehhh ... shame on me, didn't read the initial description of this PR 🙈